src/content/history/index.md

```diff
@@ -24,6 +24,32 @@ These rule changes may create a temporary split in the network. New blocks could
 
 ## 2021 {#2021}
 
+### (_In Progress_) London {#london}
+
+The London upgrade is scheduled to go live on Ethereum in July 2021. It will introduce [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), which reforms the transaction fee market, along with changes to how gas refunds are handled and the [Ice Age](https://ethereum.org/en/glossary/#ice-age) schedule.
+
+- [Read the London upgrade specification](https://github.com/ethereum/eth1.0-specs/blob/master/network-upgrades/mainnet-upgrades/london.md)
+
+<ExpandableCard title="London EIPs" contentPreview="Improvements included in this upgrade.">
+
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) – _improves the transaction fee market_
+- [EIP-3198](https://eips.ethereum.org/EIPS/eip-3198) – _returns the `BASEFEE` from a block_
+- [EIP-3238](https://eips.ethereum.org/EIPS/eip-3238) – _delays the Ice Age_
+
+_Note: more EIPs may be added to the upgrade over the comming weeks, and the EIPs above may still be modified prior to the upgrade being deployed._
+
+</ExpandableCard>
+
+---
+
+### (_In Progress_) Altair {#altair}
+
+The Altair upgrade is the first scheduled upgrade for the Beacon Chain. It is expected to go live in 2021. It will add support for "sync committees", which can enable light clients, and will bring inactivity and slashing penalties up to their full values.
```

Comment:
link to beacon chain page if possible `[Beacon Chain](/eth2/beacon-chain)`

Reply:
Fixed, thanks @ryancreatescopy